Plan/src/main/java/com/djrapitops/plan/system/tasks/SpongeTaskSystem.java

```diff
@@ -10,6 +14,20 @@
     public SpongeTaskSystem(PlanSponge plugin) {
         super(plugin, new SpongeTPSCountTimer(plugin));
     }
+    
+    @Override
+    public void enable() {
+        super.enable();
+        try {
+            PingCountTimerSponge pingCountTimer = new PingCountTimerSponge();
+            Sponge.getEventManager().registerListeners(plugin, pingCountTimer);
+            long startDelay = TimeAmount.SECOND.ticks() * (long) Settings.PING_SERVER_ENABLE_DELAY.getNumber();
+            RunnableFactory.createNew("PingCountTimer", pingCountTimer)
+                    .runTaskTimer(startDelay, PingCountTimerSponge.PING_INTERVAL);
+        } catch (ExceptionInInitializerError | NoClassDefFoundError ignore) {
+            // Running CraftBukkit
```

Comment:
This exception is a leftover from the copied class. The exception there is caused by Reflection using spigot methods, which are unavailable when running CraftBukkit.

This try catch can be removed.